src/services/theGraph.js

```diff
@@ -29,6 +43,6 @@ export const callSubgraph = (subgraphName: string, query: string) => {
     .then(({ data: response }) => response.data)
     .catch((error) => {
       reportLog(`The Graph subgraph "${subgraphName}" API call failed`, { error, query });
```

Comment:
Okay i can see what has happened here (and i did suspect that you may do this). I'll resolve the comments above asking to report to Sentry.